cmake/legacy.cmake

```diff
@@ -295,6 +295,8 @@ if(PACKAGE_SET STREQUAL full)
     URL https://root.cern/download/root_v${root_version}.source.tar.gz
     URL_HASH SHA256=d02f224b4908c814a99648782b927c353d44db79dea2cadea86138c1afc23ae9
     ${CMAKE_DEFAULT_ARGS} CMAKE_ARGS
+      "-Dbuiltin_gsl=ON"
```

Comment:
Hm, `gsl` is listed as a prerequisite: https://github.com/FairRootGroup/FairSoft/blob/master/legacy/setup-macos.sh. Does this not work any more?